code/game/area/areas/ruins/space.dm

```diff
@@ -543,3 +543,41 @@
 	name = "Syndicate Laboratory 4071 Surrounding Area"
 	icon_state = "dk_yellow"
 	color = "#26773a88"
+
+//Corporate Mining Ruin
```

Comment:
You REALLY don't need this many areas. Please cut down or use proper generics for the planet it's on.